app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -100,6 +100,8 @@ const validate = createValidator({
 });
 
 const MeetingEditor = () => {
+  const [formKey, setFormKey] = useState(0);
```

Comment:
This feels like a little bit of a dirty way to make sure the element renders the new text.. is it really necessary? And if it is, could you change it to something like `reportFieldKey` as it's not really a key for the form?

Reply:
Okay I tried to find other ways to do it and it seemed like such a hassle.. renaming the variable would still be nice though

Reply:
Renamed to formDirtyFlag